src/common/config.cc

```diff
@@ -667,9 +667,6 @@ int md_config_t::parse_argv(ConfigValues& values,
     else if (ceph_argparse_flag(args, i, "--no-mon-config", (char*)NULL)) {
       values.no_mon_config = true;
     }
-    else if (ceph_argparse_flag(args, i, "--log-early", (char*)NULL)) {
```

Comment:
@changchengx could you elaborated a little bit the rationale of this change in the commit message?

Reply:
@changchengx ping?

Reply:
@tchaikov 
Sorry for the late reply. I didn't get the email notification about this comment.
Do I need to refine the commit message?

Reply:
@changchengx yeah, please.

Reply:
@tchaikov Done